src/Fantomas/Fantomas.fsproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
```

Comment:
This should already be enabled by https://github.com/fsprojects/fantomas/blob/main/Directory.Build.props#L36

Reply:
✔️